src/components/home/modals/Receive.vue

```diff
@@ -52,10 +52,16 @@
                       class="input-t"
                       type="number"
                       v-model="amount"
+                      aria-describedby="inputLiveFeedback"
+                      :state="isWrongFormat(amount)"
                       min="0">
         </b-form-input>
+        <b-form-invalid-feedback id="inputLiveFeedback">
+          The number in this field is invalid. It can include a maximum of 8 digits after the decimal point.
```

Comment:
change this word to "The number in this field is invalid. The minimum unit of amount is 0.00000001."